pulsar-broker/src/main/java/org/apache/pulsar/broker/service/plugin/EntryFilterWithClassLoader.java

```diff
@@ -18,12 +18,15 @@
  */
 package org.apache.pulsar.broker.service.plugin;
 
+import com.google.common.annotations.VisibleForTesting;
 import java.io.IOException;
+import lombok.ToString;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.bookkeeper.mledger.Entry;
 import org.apache.pulsar.common.nar.NarClassLoader;
 
 @Slf4j
+@ToString
 public class EntryFilterWithClassLoader implements EntryFilter {
```

Comment:
do we still need this class ?

Reply:
yes, it's the implementation of EntryFilter